### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by crashzk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
+3. Go to (Website): AdminCP -> Web Panel -> **Themes**;
+4. Select theme **SourceBans++ Fluent Design Theme Edition**;
 
 ## Update 🛠️
-
 Reupload all files from repository.
 
-## Graphics 📷
-
+## Previews 📷
```
Comment (aXenDeveloper) [inline README.md:22]:
```suggestion
## Screenshots 📷
```

**Review by aXenDeveloper — COMMENTED** [review]
What are these changes for?

**Comment by crashzk** [comment]
It would be more aesthetically these changes even.

**Review by aXenDeveloper — APPROVED** [review]
I'm using `prettier` to format the code.

**Pull request merged**